pkg/mounter/oss/ossfs2.go

```diff
@@ -156,15 +156,35 @@ func (f *fuseOssfs2) buildPodSpec(c *utils.FusePodContext, target string) (spec
 	return
 }
 
+const (
+	KeyLogLevel = "log_level"
+	KeyLogDir   = "log_dir"
+)
+
 func (f *fuseOssfs2) AddDefaultMountOptions(options []string) []string {
-	alreadySet := false
+
+	defaultOSSFSOptions := os.Getenv("DEFAULT_OSSFS2_OPTIONS")
+	if defaultOSSFSOptions != "" {
+		optList := strings.Split(defaultOSSFSOptions, ",")
+		for _, opt := range optList {
+			options = append(options, strings.TrimSpace(opt))
+		}
+	}
+
+	tm := map[string]string{}
 	for _, option := range options {
-		if strings.Contains(option, "log_level") {
-			alreadySet = true
-			break
+		if option == "" {
+			continue
+		}
+		parts := strings.SplitN(option, "=", 2)
+		if len(parts) == 1 {
+			parts = append(parts, "")
 		}
+		tm[parts[0]] = parts[1]
```

Comment:
```suggestion
		k, v, _ := strings.Cut(option, "=")
		tm[k] = v
```
But I think we should have a unified place to do the parsing. Maybe `AddDefaultMountOptions` can just return all the defaults, and have another func `DeduplicateOptions` to parse the options and remove duplications? Mount options supports escaping with `\` actually. It is complicated.

Reply:
I'm not sure if another func is needed to split the defaultMountOptions retrieving and duplications removing. It together works as `AddDefaultMountOptions ` and it's meaningful. Maybe we can consider it if more default options need to be handled.

Reply:
We need to list the default options here in comments